src/components/Lists/index.js

```diff
@@ -125,20 +134,23 @@ function Lists({
       is_private: privacy,
       accounts: accountsMap,
     };
-
     try {
       await fetchJson("/api/lists", null, {
         method: "POST",
         body: JSON.stringify(payload),
       });
 
+      setLoading(true);
```

Comment:
I think this should come at the very top of the function.. before the await function